easybuild/easyblocks/b/blis.py

```diff
@@ -0,0 +1,52 @@
+"""
+Easybuild support for building and installing BLIS, implemented as an easyblock
+
+@author: Pua Cheng Xuan Frederick (National University of Singapore)
+"""
+from easybuild.easyblocks.generic.configuremake import ConfigureMake
+from easybuild.tools.systemtools import get_shared_lib_ext
+from easybuild.framework.easyconfig import CUSTOM
+from easybuild.tools.systemtools import get_cpu_arch_name
+
+
+class EB_BLIS(ConfigureMake):
+    """Custom easyblock for BLIS"""
+
+    @staticmethod
+    def extra_options():
+        """Custom easyconfig parameters for BLIS easyblock."""
+        extra_vars = {
+            'multi_threading_type': ["openmp", "Type of multithreading to use", CUSTOM],
+            'cblas_enable': [True, "Enable CBLAS", CUSTOM],
+            'shared_enable': [True, "Enable builing shared library", CUSTOM],
```

Comment:
I think `enable_cblas` makes more sense? Also we like to keep these sorted alphabetically:

```suggestion
            'enable_cblas': [True, "Enable CBLAS", CUSTOM],
            'enable_shared': [True, "Enable builing shared library", CUSTOM],
            'multi_threading_type': ["openmp", "Type of multithreading to use", CUSTOM],
```